apps/evans/components/404/NotFoundHeroBlock.tsx

```diff
@@ -1,53 +1,51 @@
 import React from 'react';
+import NextBgImage, { bgColor } from 'next-bg-image';
 import { t } from '@friends-library/locale';
 import Dual from '../core/Dual';
 import Button from '../core/Button';
-import BackgroundImage from '../core/BackgroundImage';
 import { LANG } from '@/lib/env';
 import SheepBg from '@/public/images/sheep.jpg';
 
 const NotFoundHeroBlock: React.FC = () => (
-  <BackgroundImage
-    src={SheepBg}
-    fit="cover"
-    position="[object-position:center_80%]"
-    className="text-center text-white"
+  <NextBgImage
+    className="text-center text-white px-10 py-20 sm:px-16 sm:py-24 md:py-24 xl:py-32"
+    src={[bgColor(`rgba(0, 0, 0, 0.38)`), SheepBg]}
```

Comment:
here's an example of using the `bgColor` helper component from `next-bg-image`

Reply:
Yeah that's really nice 👍